### ISSUE TITLE
Remove fixed containerd issue?

### ISSUE THREAD

**Issue by enrichman** [epinio/docs#35]
Going through the doc I've found this troubleshooting section:

https://github.com/epinio/docs/blob/30e3c0e532f34fbd31e47ccccdd2838260bfbc33/src/installation/system_requirements.md?plain=1#L29-L33

> ### Troubleshooting
> 
> While Kubernetes 1.22 is supported there is an issue when the container runtime is `containerd > 1.5.6`.
> The [pack cli](https://github.com/buildpacks/pack) is placing too much information into the
> image layers. The relevant issue is: https://github.com/paketo-buildpacks/full-builder/issues/415

The referenced issue is still open but it seems that it has been resolved.

The [containerd issue](https://github.com/containerd/containerd/issues/6123) was [fixed](https://github.com/containerd/containerd/pull/6124) and added to the [v1.5.8](https://github.com/containerd/containerd/releases/tag/v1.5.8).

Since containerd was updated in the [v1.22.4](https://github.com/k3s-io/k3s/blob/v1.22.4%2Bk3s1/go.mod#L12) of k3s I guess we can assume that it's fixed and remove the troubleshooting section, or maybe reference to update to the 1.22.4, and update and close the https://github.com/paketo-buildpacks/full-builder/issues/415 as well.

/cc @jimmykarily

**Comment by jimmykarily** [comment]
@enrichman this is right. I think we should better mention the update to `1.22.4` for a while in the docs. At least until older versions are no longer supported.
